Modelica/Fluid/Examples/HeatExchanger.mo

```diff
@@ -169,7 +169,7 @@ package HeatExchanger "Demo of a heat exchanger model"
       //Initialization pipe 1
       parameter SI.Temperature Twall_start "Start value of wall temperature"
                                                                             annotation(Dialog(tab="Initialization", group="Wall"));
-      parameter SI.Temperature dT "Start value for pipe_1.T - pipe_2.T"
+      parameter SI.TemperatureDifference dT "Start value for pipe_1.T - pipe_2.T"
```

Comment:
I am not sure if this change really belongs here since it seemss unrelaed to homotopy.

Reply:
This is somehow related, since these parameters are needed for initialization and I noticed bogus values on the parameter input dialogue when trying to fix this issue. I was short on time and couldn't open another branch. I hope you don't mind. Feel free to put this in a different PR if you want.

Reply:
@beutlich Do you want the change to `TemperatureDifference` removed from this PR so that we can proceeed?

Reply:
If you want to remove it, please re-add it with another PR, since it leads to an incorrect visualization on the parameter input windows (-272.15 °C instead of -1°C). Thanks!